multiple-select.js

```diff
@@ -222,9 +222,30 @@
                 });
             }
 
+            function isChild(p, c) {
+                if (!c || p == c) return false;
+                while (c) {
+                    c = c.parentNode;
+                    if (p==c) {
+                       return true;
+                    }
+                }
+                return false;
```

Comment:
or: https://api.jquery.com/jQuery.contains/ since jQuery is already a hard dependency in lila.